include/bluetooth/gatt.h

```diff
@@ -152,6 +152,14 @@ struct bt_gatt_attr {
 	u8_t			perm;
 };
 
+/** @brief GATT Service structure */
+struct bt_gatt_service_static {
+	/** Service Attributes */
+	const struct bt_gatt_attr *attrs;
+	/** Service Attribute count */
+	size_t			attr_count;
```

Comment:
minor: use spaces if you want to align these (not sure it improves readability though due to the embedded code comments)

Reply:
gatt.h is using tabs though, is this something we need to change accross the file? I though the coding style recommended tabs to do this type of aligments.

Reply:
> gatt.h is using tabs though, is this something we need to change accross the file? I though the coding style recommended tabs to do this type of aligments.

@Vudentz ok, I didn't realise that. Then that could be a separate cleanup PR at some point in the future. It was actually not only the tabs that bothered me but also GitHub renders it as if these are not aligned at all (even though they might be if I opened this up in an editor)